DFSNotifierResourcesDeployment/modules/configurationstore/appConfiguration.bicep

```diff
@@ -0,0 +1,35 @@
+param featureName string
+@allowed([
+  'westus2'
+  'southeastasia'
+  'westus'
+  'eastus'
+])
+param deploymentLocation string
+@allowed([
+  'ppe'
+  'prod'
+  'perf'
+])
+param environment string
+param tagsObject object
+param shortLocation string = ''
+
+var appConfigurationName = environment != 'prod' ? 'dfsn1-${featureName}-configuration-${empty(shortLocation) ? deploymentLocation : shortLocation}-${environment}' : 'dfsn-${featureName}-configuration-${empty(shortLocation) ? deploymentLocation : shortLocation}'
```

Comment:
Why is it dfsn1, I can see dfsn in multiple places ?

Reply:
resolved